project/plugins.sbt

```diff
@@ -6,6 +6,8 @@ resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "1.16.0")
 
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
```

Comment:
A  nice handy plugin for visualising project dependencies (and what they depend on)

I can remove this if required

Reply:
Those type of utility plugin is nice to have in user sbt profile
```
[lukasz.dubiel@localhost customs-declare-exports-frontend]$ cat ~/.sbt/0.13/plugins/dependecy-graph.sbt 
addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
```
then all of them would be available in every project